### ISSUE TITLE
Phone field validation

### ISSUE THREAD

**Issue by umkasanki** [verbb/formie#566]
**Description**
After updating to version 1.4.15 from 1.4.13, phone field validation of Switzerland phone numbers stopped working correctly.

I have some real phones
+41 079 546 9842
+41 22 767 6111
+41 44 563 6465
+41 62 287 4141
These number pass validation successfully on Formie 1.4.13 but not in 1.4.15 

**Steps to reproduce**
1. Add phone field
2. Check Switzerland numbers

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Ajax on typing
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.4.15
- Craft version: 3.7.11
- Multi-site: Yes 

**Additional context**

**Comment by engram-design** [comment]
Just wanted to confirm the version number you've reported is correct at 1.4.5 which was earlier in the year?

**Comment by umkasanki** [comment]
Ahh, Sorry.
1.4.13 - Switzerland phones validation works correctly
1.4.15 - Not

**Comment by engram-design** [comment]
Hmm, that's strange, those numbers are validating for me. Can you confirm your field settings?

**Comment by umkasanki** [comment]
> 
> 
> Hmm, that's strange, those numbers are validating for me. Can you confirm your field settings?

![изображение](https://user-images.githubusercontent.com/496132/132326978-1b061567-11a8-425d-b09a-9371aaed7781.png)

Sample:
https://cms.dev.bkbn.com/en/contact

**Comment by umkasanki** [comment]
It seems validation error occupies then Allowed Countries list not contains Default country

**Comment by engram-design** [comment]
Correct, it'll throw an error if the default value is not in the allowed countries. Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.15",
  "...": "..."
}
```

Then run `composer update`.

**Comment by umkasanki** [comment]
Hmm. Right you're. I've updated plugin again and now it working fine.
Sorry for disturb

**Comment by umkasanki** [comment]
Thanks Josh!

**Comment by engram-design** [comment]
Fixed in [1.4.16](https://github.com/verbb/formie/releases/tag/1.4.16)
